android/app/src/main/java/app/organicmaps/search/SearchResult.java

```diff
@@ -141,4 +142,37 @@ public Spannable getFormattedTitle(@NonNull Context context)
 
     return builder;
   }
+
+  public String getAddress(@NonNull Context context)
```

Comment:
Why is this function needed?

Reply:
There is no necessity but to make it look good i did that 
Can remove it.

Reply:
If you want a clearer code, it can be done without using a function, for example:
```
final String address = description.region;
```

But I don't see much value in it.